src/test/java/com/google/sps/servlets/CaptionTest.java

```diff
@@ -63,7 +63,9 @@ public void init() {
   @Test
   public void properXmlReturnsConcatenatedString() throws IOException {
     InputStream stream = new ByteArrayInputStream(PROPER_XML.getBytes());
-    when(captionServiceMock.fetchStream(eq(VIDEO_ID), contains(LANG_EN))).thenReturn(stream);
+    when(captionServiceMock.fetchStream(eq(VIDEO_ID), eq(LANG_EN))).thenReturn(stream);
```

Comment:
I might be wrong but I believe you only use eq() when other parameters use argument matchers, so I don't think it's necessary here

Reply:
Nice catch! it is indeed unnecessary (i replaced `contains` with `eq` from the original setup, as you may tell)